app/jmeter/jsd.jmx

```diff
@@ -4440,6 +4440,531 @@ vars.put(&quot;g_comment&quot;, comment.toString())
               <hashTree/>
             </hashTree>
           </hashTree>
+          <TransactionController guiclass="TransactionControllerGui" testclass="TransactionController" testname="jmeter_agent_created_vs_resolved_report" enabled="true">
```

Comment:
please rename to `jmeter_agent_view_created_vs_resolved_report`

Reply:
done